crowbar_framework/config/environments/production.rb

```diff
@@ -41,10 +41,10 @@
 
   config.assets.debug = false
   config.assets.raise_runtime_errors = true
-  config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = :closure
   config.assets.css_compressor = :sass
-  config.assets.compile = true
-  config.assets.digest = true
```

Comment:
As you can see in the commit message for that since rails 3 it is not possible anymore to disable digest. So we need to find a different solution for the static pages...